[CI] DocsClientYamlTestSuiteIT test {yaml=reference/data-streams/failure-store-recipes/line_201} failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #11559 / windows-2022_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/4ihnhln6cnkak)
- [elasticsearch-periodic-platform-support #11559 / windows-2025_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/ivynmli4hklui)
- [elasticsearch-periodic-platform-support #11559 / windows-2019_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/zdxms642ytunw)
- [elasticsearch-periodic-platform-support #11559 / windows-2022_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/ixxmbbwsqebym)
- [elasticsearch-periodic-platform-support #11555 / windows-2025_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/vuxispvznw2he)
- [elasticsearch-periodic-platform-support #11555 / windows-2019_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/uor7n7lqs2yas)
- [elasticsearch-periodic-platform-support #11555 / windows-2022_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/2vlexbzd5algc)
- [elasticsearch-periodic-platform-support #11555 / windows-2019_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/cpox3bc5vcmh6)
- [elasticsearch-periodic-platform-support #11555 / windows-2025_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/dw6xbsbewxbos)
- [elasticsearch-periodic-platform-support #11555 / windows-2022_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/q7ufnaengfqvo)

**Reproduction Line:**
```
gradlew ":docs:yamlRestTest" --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/data-streams/failure-store-recipes/line_201}" -Dtests.seed=A3B49284A61F8A71 -Dtests.locale=ie-Latn-EE -Dtests.timezone=America/Argentina/Catamarca -Druntime.java=25
```

**Applicable branches:**
8.19

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Byaml%3Dreference%2Fdata-streams%2Ffailure-store-recipes%2Fline_201%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [reference/data-streams/failure-store-recipes:331]: Expected a map containing
  _shards: a map containing
      total: <1>
     failed: <0>
 successful: <1>
    skipped: <0>
     hits: a map containing
       hits: a list containing
            0: a map containing
         _index: ".fs-my-datastream-ingest-2025.12.16-000001"
        _source: a map containing
       @timestamp: "2025-12-16T14:43:33.246Z"
         document: a map containing
              index: "my-datastream-ingest"
             source: a map containing
           @timestamp: "2025-04-21T00:00:00Z"
         counter_name: "test"
            error: a map containing
           pipeline: "complicated-processor"
     processor_type: "set"
      processor_tag: "copy to new counter again"
     pipeline_trace: a list containing
                    0: "complicated-processor"
        stack_trace: expected "j.l.IllegalArgumentException: field [counter] not present as part of path [counter]\n\ta
[truncated]
```

**Issue Reasons:**
- [8.19] 4 consecutive failures in test test {yaml=reference/data-streams/failure-store-recipes/line_201}
- [8.19] 11 consecutive failures in step windows-2022_checkpart1_platform-support-windows
- [8.19] 10 consecutive failures in step windows-2025_checkpart1_platform-support-windows
- [8.19] 10 consecutive failures in step windows-2019_checkpart1_platform-support-windows
- [8.19] 32 failures in test test {yaml=reference/data-streams/failure-store-recipes/line_201} (12.7% fail rate in 251 executions)
- [8.19] 11 failures in step windows-2022_checkpart1_platform-support-windows (100.0% fail rate in 11 executions)
- [8.19] 10 failures in step windows-2025_checkpart1_platform-support-windows (100.0% fail rate in 10 executions)
- [8.19] 10 failures in step windows-2019_checkpart1_platform-support-windows (100.0% fail rate in 10 executions)
- [8.19] 7 failures in pipeline elasticsearch-periodic-platform-support (100.0% fail rate in 7 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/docs/reference/data-streams/failure-store-recipes.asciidoc
+++ b/docs/reference/data-streams/failure-store-recipes.asciidoc
@@ -107,6 +107,7 @@ GET my-datastream-ingest::failures/_search
 // TESTRESPONSE[s/"_index": ".fs-my-datastream-ingest-2025.05.09-000001"/"_index": $body.hits.hits.0._index/]
 // TESTRESPONSE[s/"_id": "F3S3s5YBwrYNjPmayMr9"/"_id": $body.hits.hits.0._id/]
 // TESTRESPONSE[s/"@timestamp": "2025-05-09T06:24:48.381Z"/"@timestamp": $body.hits.hits.0._source.@timestamp/]
+// TESTRESPONSE[s/"stack_trace": ".*"/"stack_trace": $body.hits.hits.0._source.error.stack_trace/]
 <1> When an ingest pipeline fails, the document stored is what was originally sent to the cluster.
 <2> The important information that we failed to find was originally present in the document.
 <3> The info field was not present when the failure occurred.
@@ -341,6 +342,7 @@ GET my-datastream-ingest::failures/_search
 // TESTRESPONSE[s/"_index": ".fs-my-datastream-ingest-2025.05.09-000001"/"_index": $body.hits.hits.0._index/]
 // TESTRESPONSE[s/"_id": "HnTJs5YBwrYNjPmaFcri"/"_id": $body.hits.hits.0._id/]
 // TESTRESPONSE[s/"@timestamp": "2025-05-09T06:41:24.775Z"/"@timestamp": $body.hits.hits.0._source.@timestamp/]
+// TESTRESPONSE[s/"stack_trace": ".*"/"stack_trace": $body.hits.hits.0._source.error.stack_trace/]
 <1> Helpful, but which set processor on the pipeline could it be?
 <2> The tag of the exact processor that the document failed on.
 

